database/postgres/definition.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: apiextensions.crossplane.io/v1
+kind: CompositeResourceDefinition
+metadata:
+  name: xpostgresqlinstances.azure.platformref.crossplane.io
+  annotations:
+    upbound.io/ui-schema: |-
+      ---
+      configSections:
+      - title: Database Size
+        description: Enter information to size your database
+        items:
+        - name: storageGB
+          controlType: singleInput
+          type: integer
+          path: ".spec.parameters.storageGB"
+          title: Storage (GB)
+          description: The size in GB for database storage
+          default: 5
+          validation:
+          - minimum: 1
+          - maximum: 500
+          - required: true
+        - name: networkRef
+          controlType: singleInput
+          type: string
+          path: ".spec.parameters.networkRef.id"
```

Comment:
i think networkRef isn't being used for this resource either, so it could be removed from the UI metadata